parts/linux/cloud-init/artifacts/kubelet.service

```diff
@@ -32,8 +32,9 @@ ExecStart=/usr/local/bin/kubelet \
         --enable-server \
         --node-labels="${KUBELET_NODE_LABELS}" \
         --v=2 {{if NeedsContainerd}}--container-runtime=remote --runtime-request-timeout=15m --container-runtime-endpoint=unix:///run/containerd/containerd.sock{{end}} \
-        --volume-plugin-dir=/etc/kubernetes/volumeplugins \
-        $KUBELET_CONFIG \
+        --volume-plugin-dir=/etc/kubernetes/volumeplugins \{{if IsDynamicKubeletSupported}}
+        --config /etc/default/kubeletconfig.json)} \{{end}}
```

Comment:
I just tried the hyphen syntax ({{- If ...}}) and it works. So you still put the if statement in a new line with a prefix hyphen

Reply:
changed to use {{- if